app/src/main/java/com/willowtree/vocable/settings/EditPresetsFragment.kt

```diff
@@ -14,13 +14,17 @@ import androidx.viewpager2.widget.ViewPager2
 import com.willowtree.vocable.BaseFragment
 import com.willowtree.vocable.R
 import com.willowtree.vocable.customviews.PointerListener
+import com.willowtree.vocable.databinding.EditPhrasesActionButtonBinding
 import com.willowtree.vocable.databinding.FragmentEditPresetsBinding
+import com.willowtree.vocable.databinding.PhraseEditLayoutBinding
 import com.willowtree.vocable.room.Phrase
+import org.koin.android.ext.android.bind
 import java.lang.Math.ceil
 
 class EditPresetsFragment : BaseFragment() {
 
     private var binding: FragmentEditPresetsBinding? = null
+    private var bindingEditPhrases: PhraseEditLayoutBinding? = null
```

Comment:
What is this second biding used for?

Reply:
I think I was using it for something else, but it's not used anymore. Thank you!!!